CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeService jokeService;
+        private IJokeOutput jokeOutput;
```

Comment:
Field should be underscore PascalCase
```suggestion
        private IJokeService _JokeService;
        private IJokeOutput _JokeOutput;
```